test_scripts/Smoke/API/009_DeleteInteractionChoiceSet_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,130 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: DeleteInteractionChoiceSet
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [DeleteInteractionChoiceSet] SUCCESS choiceSet removal
+--
+-- Description:
+-- Mobile application sends valid DeleteInteractionChoiceSet request to SDL
+-- and interactionChoiceSet with <interactionChoiceSetID> was successfully
+-- removed on SDL and HMI for the application.
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and on SDL
+-- c. appID is in Background, Full or Limited HMI level
+-- d. Choice set with <interactionChoiceSetID> is created
+
+-- Steps:
+-- appID requests DeleteInteractionChoiceSet request with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if VR interface is available on HMI
+-- SDL checks if DeleteInteractionChoiceSet is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the VR.DeleteCommand with allowed parameters to HMI
+-- SDL receives successful responses to corresponding VR.DeleteCommand from HMI
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local putFileParams = {
+	requestParams = {
+	    syncFileName = 'icon.png',
+	    fileType = "GRAPHIC_PNG",
+	    persistentFile = false,
+	    systemFile = false
+	},
+	filePath = "files/icon.png"
+}
+
+local createRequestParams = {
+	interactionChoiceSetID = 1001,
+	choiceSet = {
+		{
+			choiceID = 1001,
+			menuName ="Choice1001",
+			vrCommands = {
+				"Choice1001"
+			},
+			image = {
+				value ="icon.png",
+				imageType ="DYNAMIC"
+			}
+		}
+	}
+}
+
+local createResponseVrParams = {
+	cmdID = createRequestParams.interactionChoiceSetID,
+	type = "Choice",
+	vrCommands = createRequestParams.vrCommands
+}
+
+local createAllParams = {
+	requestParams = createRequestParams,
+	responseVrParams = createResponseVrParams
+}
+
+local deleteRequestParams = {
+	interactionChoiceSetID = createRequestParams.interactionChoiceSetID
+}
+
+local deleteResponseVrParams = {
+	cmdID = createRequestParams.interactionChoiceSetID,
+	type = "Choice"
```

Comment:
add grammarID

Reply:
@GetmanetsIrina done in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1716/commits/7822eee4408704d77053a8a725acc23c5f5b4ea6